cpp/examples/ssl.cpp

```diff
@@ -165,16 +165,19 @@ int main(int argc, char **argv) {
         opts.parse();
 
         size_t sz = cert_directory.size();
-        if (sz && cert_directory[sz -1] != '/')
-            cert_directory.append("/");
-        else cert_directory = "ssl-certs/";
+        if (sz) {
+           if (cert_directory[sz -1] != '/')
+               cert_directory.append("/");
+        } else cert_directory = "ssl-certs/";
```

Comment:
Bad indentation (yes I know the original was bad too!) please use explicit '{' and '}' and indent. I'd also prefer the else to be indented in a '{' '}' block too, but that's not bad indentation (IMO).